### PR TITLE
WS-5350 Updated Link and Pagination Selector Information

### PR DESCRIPTION
**Pull request by karinaMeldere**
Added another link type to "Link selector" page and reformatted the "Pagination selector" page.

### CODE REVIEW

**Review by einarsozols — APPROVED** [review]

**Review by ivarsweb — APPROVED** [review]

**Comment by martinsbalodis** [comment]
waiting for Ivars second review

File: docs/Selectors/Link selector.md
Comment (ivarsweb) [inline docs/Selectors/Link selector.md]: All Link Selector types except for 'Link from any script' only allow selecting <a> elements using the point-and-click interface
although other elements can be selected by manually entering the CSS selector value as a selector.

The 'Link from any script' type allows any element to be selected using the point-and-click interface.

**Review by ivarsweb — CHANGES_REQUESTED** [review]

**Review by ivarsweb — APPROVED** [review]

**Review by martinsbalodis — APPROVED** [review]

**Pull request merged**
